src/Service/PageTitle.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\Service;
+
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class PageTitle
+{
+    public function __construct(
+        private readonly BreadcrumbTrail $breadcrumbTrail,
+        private readonly Fallbacks $fallbacks,
+        private readonly TranslatorInterface $translator,
+    ) {
+    }
+
+    public function getTitle(): string
+    {
+        $breadcrumbs = $this->breadcrumbTrail->all();
+        $breadcrumbs =array_reverse($breadcrumbs);
```

Comment:
Code styling, missing space after `=`

Reply:
Why not in 1 line?

Reply:
fixed c789ac4076c1bdf04c69581bf21793e210e2589f